alf/trainers/off_policy_trainer.py

```diff
@@ -85,8 +85,14 @@ def __init__(self, config):
         self._driver_started = False
 
     def _init_driver(self):
-        for _ in range(1, self._config.num_envs):
-            self._create_environment()
+        for i in range(1, self._config.num_envs):
+            # [self._random_seed, self._random_seed + batch_size) has been used
+            # in policy_trainer.py
+            if self._random_seed is None:
+                self._create_environment()
+            else:
+                self._create_environment(
+                    random_seed=self._random_seed + i * common._env.batch_size)
```

Comment:
Why was this bug not caught by CI?

Reply:
Maybe because this is only async training with multiple envs?

Reply:
Perhaps we should add a test for it. We can do it in another PR.